frontend/src/metabase/dashboard/components/DashCard/DashCard.tsx

```diff
@@ -160,8 +162,10 @@ function DashCardInner({
   }, [cards, dashcard.id, dashcardData, slowCards]);
 
   const isLoading = useMemo(
-    () => isDashcardLoading(dashcard, dashcardData),
-    [dashcard, dashcardData],
+    () =>
+      isDashcardLoading(dashcard, dashcardData) ||
```

Comment:
Would it make sense to update `isDashcardLoading` to include `loadingDashcardIds` check? Otherwise the function is somewhat invalid because it doesn't do what it promises without an extra check outside.

Reply:
Updated.